app/root.tsx

```diff
@@ -1,5 +1,5 @@
 import {
-	json,
+	data as dataResponse,
```

Comment:
I think I'd prefer to just import this as `data` everywhere. I realize this means we get variable shadowing in some places, but I don't see that as being a practical concern. Those variables could (probably should) be changed.